noetic/distribution.yaml

```diff
@@ -11253,6 +11253,26 @@ repositories:
       url: https://github.com/ros-industrial/ur_msgs.git
       version: melodic-devel
     status: maintained
+  ur_robot_driver:
+    doc:
+      type: git
+      url: https://github.com/UniversalRobots/Universal_Robots_ROS_Driver.git
+      version: master
+    release:
+      packages:
+      - controller_stopper
+      - ur_calibration
+      - ur_dashboard_msgs
+      - ur_robot_driver
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/UniversalRobots/Universal_Robots_ROS_Driver-release.git
+      version: 2.1.0-1
+    source:
+      type: git
+      url: https://github.com/UniversalRobots/Universal_Robots_ROS_Driver.git
```

Comment:
@fmauch I see a few LICENSE files in the source repository but none at the root. Would you mind adding a copy (or relocating one of the existing ones)?

Reply:
sure, that should not be a problem.